src_Core/ISA/ISA_Decls_Priv_S.bsv

```diff
@@ -409,17 +418,19 @@ function Tuple2#(Bool,Exc_Code) is_pte_fault
 
    Bool access_cap_ok = (   (! capability)
 			 || access_load
-			 || (access_store && (pte_StoreCap == 1'b1)));
```

Comment:
Missing ")"

Reply:
Arg, I swear I fixed that.  Must have mangled `git` somehow. :\

Reply:
Ah, the rookie mistake: `add` is not `commit`.